### ISSUE TITLE
Change Color of DefaultNumericAxis TickMark labels

### ISSUE THREAD

**Issue by carbotaniuman** [GSI-CS-CO/chart-fx#157]
I'm trying to accomplish a dark theme style for my graph, and the current barrier to that is the fact that the TickMarks in the graph are black, like so:

![Capture](https://user-images.githubusercontent.com/41451839/79594531-c49e7e00-80a2-11ea-8436-f2e35c06929f.PNG)

I've tried looking at the `de.gsi.chart.axes.spi.TickMark` class, but it does not have a styleclass added. Is there any way I can style these tickmark labels?

**Comment by RalphSteinhagen** [comment]
Hi @carbotaniuman thanks for your interest and welcome to the growing Chart-fx community.

There are CSS as well as direct access methods to change the colour, font, stroke, stroke width etc.. programmatically. You may try:
```
 xAxis.getAxisLabel().setFill(Color.MAGENTA);
 xAxis.getMajorTickStyle().setStroke(Color.AQUAMARINE);
 xAxis.getMinorTickStyle().setStroke(Color.RED);
 xAxis.getMinorTickStyle().setStrokeWidth(1.5);
 xAxis.setTickLabelFill(Color.DARKGREEN);
```
should yield the following:
![image](https://user-images.githubusercontent.com/46007894/79669432-b1cb9c80-81bb-11ea-8e57-167ceab268a0.png)
(N.B. sorry for the intentially poor choice of colours, this was meant for testing purposes :smile: )

N.B. There was missing direct wiring between the major tick-mark font and font colour (ie. `setTickLabelFill(Paint)`must have gotten lost during one of the last refactoring/clean-up). The short-term workaround would have been to set the parent 'text-fil...'. A bit hacky and I thus pushed the fix. Should in master and released soonish.  See PR #158.

Thanks again for your interest!

**Comment by carbotaniuman** [comment]
What is the CSS selector for `xAxis.setTickLabelFill(Color.DARKGREEN);`?

**Comment by RalphSteinhagen** [comment]
The mentioned fix 82ebf46 is now merged. You may want to re-pull master and/or use the JDK11 [snapshot release](https://github.com/GSI-CS-CO/chart-fx#using-the-snapshot-repository).

Before this you would have had to use the generic `-fx-fill` (declared originally in `Shape`) tag, but now this can (again) be more specifically set for TickMarks via the setter/getter/property and/or the `-fx-tick-label-fill` CSS statement.

Hope this helps. Enjoy!

**Comment by RalphSteinhagen** [comment]
I will close this issue since the functionality/fix is merged and we did not receive any more feedback. Please feel free to reopen if you find that this still does not suit your needs.
